.kitchen.vagrant.yml

```diff
@@ -1,18 +1,27 @@
 ---
 driver:
   name: vagrant
+  driver_config:
+    http_proxy: <%= ENV['http_proxy'] || nil %>
```

Comment:
I usually set this in the global local Vagrantfile, so it works absolutely with all Vagrant and kitchen files for me:

```bash
$ cat ~/.vagrant.d/Vagrantfile
Vagrant.configure("2") do |config|
  if Vagrant.has_plugin?('vagrant-proxyconf')
    config.proxy.http     = ENV['http_proxy'] if ENV.key?('http_proxy')
    config.proxy.https    = ENV['https_proxy'] if ENV.key?('https_proxy')
    config.yum_proxy.http = ENV['https_proxy'] if ENV.key?('https_proxy')
    config.proxy.no_proxy = ENV['no_proxy'] if ENV.key?('no_proxy')
  end

  if !ENV["GLOBAL_VAGRANT_CACHIER_DISABLED"] && Vagrant.has_plugin?("vagrant-cachier")
    config.cache.scope = :box
  end
end
```